ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Follower.hs

```diff
@@ -211,29 +211,29 @@ instructionHelper ::
      -- @Maybe@.
   -> ChainDbEnv m blk
   -> m (f (ChainUpdate blk b))
-instructionHelper registry varReader blockComponent fromMaybeSTM CDB{..} = do
+instructionHelper registry varFollower blockComponent fromMaybeSTM CDB{..} = do
     -- In one transaction: check in which state we are, if in the
-    -- @ReaderInMem@ state, just call 'instructionSTM', otherwise,
-    -- return the contents of the 'ReaderInImmutableDB' state.
+    -- @FollowerInMem@ state, just call 'instructionSTM', otherwise,
+    -- return the contents of the 'FollowerInImmutableDB' state.
     inImmutableDBOrRes <- atomically $ do
       curChain <- readTVar cdbChain
-      readTVar varReader >>= \case
+      readTVar varFollower >>= \case
         -- Just return the contents of the state and end the transaction in
         -- these two cases.
-        ReaderInit
+        FollowerInit
           -> return $ Left (RollBackTo GenesisPoint, Nothing)
-        ReaderInImmutableDB rollState immIt
+        FollowerInImmutableDB rollState immIt
           -> return $ Left (rollState, Just immIt)
 
-        ReaderInMem   rollState
+        FollowerInMem   rollState
```

Comment:
I don't know why the space before `rollState` was there to begin with. Please remove it.